concent_api/core/validation.py

```diff
@@ -178,12 +178,14 @@ def validate_golem_message_signed_with_key(
 
     try:
         golem_message.verify_signature(public_key)
-    except MessageError:
-        raise Http400(
-            'There was an exception when validating if golem_message {} is signed with public key {}'.format(
+    except MessageError as exception:
+        error_message = 'There was an exception when validating if golem_message {} is signed with public key {}.'.format(
                 golem_message.TYPE,
-                public_key
-            ),
+                public_key,
+        )
+        error_message += str(exception)
```

Comment:
Do you remember your function for joining strings in messages? Use it.

Reply:
Done